ruby-challenges.rb

```diff
@@ -19,17 +25,58 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def find_sum(arr)
+    arr.reduce(:+)
+end
 
+p find_sum(nums_array1)
+p find_sum(nums_array2)
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
-# Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
-
+class Bike
+    
```

Comment:
It could be the GitHub UI, but there are a lot of extra spaces here. Just something to keep an eye on.